src/modules/emote_menu/utils/twitch-emotes.js

```diff
@@ -102,9 +116,34 @@ export async function loadTwitchEmotes() {
   const isDark = settings.get(SettingIds.DARKENED_MODE);
   const tempCategories = {};
 
-  for (const {owner, id: setId, emotes} of data.channel.self.availableEmoteSets) {
+  // so we don't rely on uniqBy https://github.com/night/betterttv/pull/4849#discussion_r711787270
+  const emoteSets = data.channel.self.availableEmoteSets;
+
+  const availableChannelEmotes = flatten(
+    data.channel.self.availableEmoteSets
+      .filter(({owner}) => owner?.id === data.user.id)
+      .map((channel) => channel.emotes)
+  );
+
+  const unavailableChannelEmotes = flatten(data.user.subscriptionProducts.map(({emotes}) => emotes))
+    .filter(({id}) => availableChannelEmotes.find(({id: emoteId}) => emoteId === id) == null)
+    .map((emote) => ({...emote, locked: true}));
+
+  if (unavailableChannelEmotes.length > 0) {
+    emoteSets.push({
+      emotes: unavailableChannelEmotes,
+      id: EmoteCategories.UNAVAILABLE_CHANNEL_EMOTES,
```

Comment:
rather than use a separate category, could we instead sort them within the same category?